package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
do we need this file? This is a Magento extension not a node module.

Reply:
Here `package.json` is a helper file for the developer to install packages needed to test their code. We could add a install command to `.travis.yml` like `npm install editorconfig-tools` but that kind of setup have been replaced once before when Composer dev requirements was handled that way e2484ce47b2fe1687b5ed6ab6fda3a8c13d804a4